INSTALL.md

```diff
@@ -59,17 +59,19 @@ To update the project from within the project's folder, you can run the followin
 git pull
 ```
 
+For a stable build it is recommended to checkout the master branch.
+
+```bash
+git checkout master
+```
+
 Navigate to the directory where you have downloaded KeePassXC and type these commands:
 
 ```
 cd directory-where-sources-live
-mkdir build
-cd build
```

Comment:
Please do not remove these steps. Yo don't want to have the build files inside the source directory. Yes, it is possible to compile without, but it's not recommended. It messes up the source folder and it may create other problems when you are starting KeePassXC from the source directory.

Reply:
Ah ok, then I did not understand how to run cmake in build. `CMakeLists.txt` needs to be present right? Or is there another option that I did not see?

Reply:
Just follow the build steps exactly as they are written.

Reply:
Maybe replace the line
```
cmake -DWITH_TESTS=OFF ...and other options - see below...
```

with 
```
cmake [CMake options, see below] ..
```

Reply:
Actually, replace it with
```
cmake -DWITH_XC_ALL=ON ..
```

Reply:
AH ok, sorry I'm not familiar with cmake but that helps.